stix2/base.py

```diff
@@ -124,11 +124,20 @@ def __len__(self):
 
     # Handle attribute access just like key access
     def __getattr__(self, name):
-        return self.get(name)
+        # If the requested attribute is not None return its value
```

Comment:
That's the problem I was experiencing. Doing `self[name]` or `self.__getitem__(name)` would generate an AttributeError because the key will not exist on the `self._inner` dictionary if there is no value set for that property. So I don't have a clear way to check "is the key there?" and "Is the value of the key not None?". For example a check like `bundle.objects is None` raises an AttributeError if "objects" is not set with the current suggested changes.